src/app/action-items/action-items.component.html

```diff
@@ -39,16 +43,17 @@
                 <div class="flexRow bodyCell inner" role="cell">{{ actionItem.completed }}%</div>
                 <div class="flexRow bodyCell inner" role="cell">{{ (actionItem.dueDate | date: 'MM/dd/yyyy') || '-' }}</div>
                 <div class="flexRow bodyCell inner" role="cell">
-                    {{ actionItem | daysLeftCounted }}
+                    {{ actionItem.dueDate | daysLeftCounted }}
                 </div>
                 <div class="flexRow bodyCell inner" role="cell">
                     <a><i class="fa fa-edit"></i></a>
                 </div>
             </div>
         </div>
     </div>
-    <div class="innerContainer detailsContainer ">
-        <app-item-details [id]="actionItemId" [item]="actionItem" [isLoadingActionItem]="isLoadingActionItem" [errorMessage]="errorMessage">
+    <div class="innerContainer detailsContainer">
+        <app-item-details [showImageWhenNoActionItem]="showImageWhenNoActionItem" [actionItemId]="actionItemId" [actionItem]="actionItem" [isLoadingActionItem]="isLoadingActionItem" [errorMessage]="errorMessage" [dialogData]="dialogData" (refreshViewAfterDeletion)="refreshViewAfterDeletion($event)"
```

Comment:
`actionItem` already contains `id` - why do you need to pass `actionItem` as well as `actionItemId`?